com.unity.cinemachine/Samples~/2D Samples/Prefabs/Simple Room.prefab

```diff
@@ -0,0 +1,842 @@
+%YAML 1.1
```

Comment:
I would take the light out of this prefab.  If I put 2 rooms in the scene, I'll get 2 lights - not good